src/app.module.ts

```diff
@@ -1,9 +1,19 @@
 import { Module } from '@nestjs/common';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { TypegooseModule } from 'nestjs-typegoose';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { getMongoConfig } from './configs/mongo.config';
+import { ContactModule } from './contact/contact.module';
+import { ListModule } from './list/list.module';
 
 @Module({
-  imports: [],
+  imports: [ConfigModule.forRoot(), 
```

Comment:
Please, after [ start from new line
Each array item should be on new line
[ 
  ContactModule, 
  ListModule,
]

Reply:
the blank line was added